code/modules/mob/living/basic/lavaland/gutlunchers/gutlunchers.dm

```diff
@@ -11,7 +11,7 @@
 	combat_mode = FALSE
 	icon_living = "gutlunch"
 	icon_dead = "gutlunch"
-	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	mob_biotypes = MOB_ORGANIC|MOB_BUG|MOB_MINING
```

Comment:
You removing MOB_BEAST from these got rid of junk hunter rounds and strongarm implants' increased damage on those mobs. They're wild animals, so it probably should stick.

Reply:
I should probably just expand the rounds/arms to include MOB_BUG, I forgot it existed. I'll do it in another PR after this one.